src/components/TeacherComponents/UnitsLearningThemeFilters/UnitsLearningThemeFilters.tsx

```diff
@@ -114,23 +112,18 @@ const UnitsLearningThemeFilters = ({
         role="radiogroup"
         $pb="inner-padding-xl2"
       >
-        {[{ slug: "all", label: "All" }, ...learningThemesMapped].map(
-          (theme) => {
-            const isChecked = activeThemeSlug === theme.slug;
-            const isFocussed = focussedThemeSlug === theme.slug;
-            return (
-              <RadioTile
-                theme={theme}
-                key={theme.slug}
-                isChecked={isChecked}
-                isFocussed={isFocussed}
-                onChange={onChange}
-                onFocus={setFocussedThemeSlug}
-                id={`${theme.slug}-${idSuffix}`}
-              />
-            );
-          },
-        )}
+        {[{ id: "all", label: "All" }, ...themeTileItems].map((theme) => {
+          const isChecked = activeThemeSlug === theme.id;
+          return (
+            <OakRadioTile
+              tileItem={theme}
```

Comment:
:surfer: not something to action now, but we should be cautious about passing non-primitives as props (particularly to components in the library). Theoretically it will make components harder to optimise later should we need to since the value is more likely to change between renders and harder to test equality. In reality React is slow anyway so it's not the end of the world. I'd usually prefer to pass each value as an individual prop when there are only a couple of things.